agent/php_minit.c

```diff
@@ -491,6 +492,18 @@ PHP_MINIT_FUNCTION(newrelic) {
    */
   nr_php_generate_internal_wrap_records();
 
+#if ZEND_MODULE_API_NO >= ZEND_8_0_X_API_NO
+  /* 
+   * The user function wraprec hashmap must be initialized before INI processing
+   * because INI processing adds wraprecs:
+   *  - newrelic.webtransaction.name.functions
+   *  - newrelic.transaction_tracer.custom
+  */
+  if (NR_SUCCESS != nr_php_user_instrument_wraprec_hashmap_init()) {
+    nrl_error(NRL_AGENT, "Failed to initialize user function instrumentation");
```

Comment:
If this happens, do we want to `goto disbad`? I am unsure of the implications of skipping `nr_php_register_ini_entries`, as all other `goto disbad`'s are afterward.